lib/mix/tasks/phx.digest.clean.ex

```diff
@@ -44,6 +44,7 @@ defmodule Mix.Tasks.Phx.Digest.Clean do
     keep = opts[:keep] || @default_keep
     all? = opts[:all] || false
 
+    Mix.Task.run("compile", args)
```

Comment:
Why this change?

Reply:
I accidentally answered in change suggestion: https://github.com/phoenixframework/phoenix/pull/5318#discussion_r1123429862